product.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require_once 'src/Services/ProductsDisplayService.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Factory/furnitureDatabaseConnector.php';
```

Comment:
```suggestion
require_once 'src/Factory/FurnitureDatabaseConnector.php';
```

Capitals for class names